assets/src/edit-story/components/colorPicker/karma/eyedropper.karma.js

```diff
@@ -77,6 +81,12 @@ describe('Eyedropper', () => {
     const imageOnCanvasRect = (
       await getCanvasElementWrapperById(imageOnCanvas.id)
     ).getBoundingClientRect();
+    // uncomment and view karma run to see how this isn't properly aligned
+    // await fixture.events.mouse.move(
+    //   imageOnCanvasRect.right - 2,
+    //   imageOnCanvasRect.top + 8
+    // );
+    // await fixture.events.sleep(10000);
```

Comment:
@merapi @barklund if you uncomment this, and run `npm run test:karma:edit-story:watch` you can see the misalignment issue I mentioned.

Any thoughts on this? seems like an issue with the eyedropper and not the test, but maybe I'm misunderstanding something. Anyway, y'all have more context here. curious to see what you think

Reply:
Found it - I can replicate the issue on one of my machines, will fix it today/tomorrow, right now I will work on a general fix for Moveable (double click/dnd issue).